.github/workflows/python-ci.yml

```diff
@@ -84,3 +84,21 @@ jobs:
     - name: Show debug logs
       if: ${{ failure() }}
       run: docker compose -f dev/docker-compose.yml logs
+
+  object-store-integration-test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python: ['3.9', '3.10', '3.11', '3.12']
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install system dependencies
+      run: sudo apt-get update && sudo apt-get install -y libkrb5-dev # for kerberos
+    - name: Install
+      run: make install
+    - name: Run object store integration tests
+      run: make test-coverage-object-store-integration
+    - name: Show debug logs
+      if: ${{ failure() }}
+      run: docker compose -f dev/docker-compose.yml logs
```

Comment:
This one is not applicable here. If we want to do this, we need to do it per docker-compose, eg `dev/docker-compose-azurite.yml`, `dev/docker-compose-gcs-server.yml`, etc.